Source/WebCore/CMakeLists.txt

```diff
@@ -2226,6 +2226,10 @@ if (USE_LIBWEBRTC)
     )
 endif ()
 
+# FIXME: This needs a more scalable solution (webkit.org/b/267080).
+if (ENABLE_INPUT_TYPE_DATE)
+    set(FEATURE_DEFINES_WITH_SPACE_SEPARATOR "${FEATURE_DEFINES_WITH_SPACE_SEPARATOR} ENABLE_DATE_AND_TIME_INPUT_TYPES")
+endif ()
 if (HAVE_OS_DARK_MODE_SUPPORT)
     set(FEATURE_DEFINES_WITH_SPACE_SEPARATOR "${FEATURE_DEFINES_WITH_SPACE_SEPARATOR} HAVE_OS_DARK_MODE_SUPPORT")
 endif ()
```

Comment:
I think the only problem here is `ENABLE_DATE_AND_TIME_INPUT_TYPES` is not a real WebKit feature because it is not declared in WebKitFeatures.cmake (or any port-specific Options*.cmake). So yeah, just add `ENABLE_DATE_AND_TIME_INPUT_TYPES` to WebKitFeatures.cmake and this problem should go away. I think. Right?

Then I would add a different FIXME above this `if (HAVE_OS_DARK_MODE_SUPPORT)` block complaining that it is messy. WebKit features don't start with `HAVE_` so it's harder to decide what to do about that one.

Reply:
I do worry a bit that properly defining it would result in a bunch of new failures, but maybe not. I guess I'll experiment a bit more tomorrow.

This should have been essentially a no-op change so it's rather weird to now be stuck.